workspaces/blackduck/.changeset/light-deers-shake.md

```diff
@@ -0,0 +1,6 @@
+---
+'@backstage-community/plugin-blackduck-backend': patch
+---
+
+Fixed bug when project name contains special char.
+Fixed wrong message typo.
```

Comment:
If you make the above change then we can delete this line. Just formats better in the changelog latter on 👍